.flowconfig

```diff
@@ -6,15 +6,16 @@
 <PROJECT_ROOT>/.*/node_modules/y18n/.*
 <PROJECT_ROOT>/node_modules/chrome-devtools-frontend/.*
 <PROJECT_ROOT>/node_modules/devtools-timeline-model/.*
+<PROJECT_ROOT>/www/node_modules/.*
 <PROJECT_ROOT>/.*/__mocks__/.*
 <PROJECT_ROOT>/.*/__tests__/.*
 
 # Ignore Docs
 <PROJECT_ROOT>/docs/.*
 <PROJECT_ROOT>/.*/docs/.*
-<PROJECT_ROOT>/www/.*
 
 [include]
+./www/node_modules/hex2rgba/
```

Comment:
That's interesting. I'm curious why we have to include the copy of `hex2rgba` in `node_modules` here, given that we declare the module below?